src/datasources/CartsDataSource.ts

```diff
@@ -16,15 +16,14 @@ export class CartsDataSource extends RESTDataSource {
   }
 
   async getCart(id) {
-    const { data } = await this.get(`/carts/${id}/`);
+    const results = await this.get(`/carts/${id}`, {
+      include: 'items'
+    });
+    var data = results.data;
+    data.included = results.included
     return data;
```

Comment:
```suggestion
    const { data, included }= await this.get(`/carts/${id}`, {
      include: 'items'
    });
    return { data, included }
```

Reply:
I like it but that won't work because it returns 
"{"data":{"id":"....
instead of 
{"id":"

Reply:
Sorry, my bad. How about this? Please feel free to revert my changes :) 
```
    const { data, included }= await this.get(`/carts/${id}`, {      include: 'items'    });    
    data.included = included
    return data
```

Reply:
perfect